src/HelperLocator.php

```diff
@@ -3,11 +3,11 @@
 
 class HelperLocator
 {
-    static public function new(Escape $escape = null) : self
+    static public function new(Escape $escape = null, $factories = []) : self
```

Comment:
```suggestion
    static public function new(Escape $escape = null, $factories = []) : staic
```

Reply:
for the inherited class

Reply:
I think your suggestion has a typo. `staic` vs `static`

Reply:
Another note probably is type hint to array. So I can have both in single commit. Thougths ?

Reply:
Sure. Updated!